src/components/vote/Vote.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import React, { useState } from "react";
+import "./vote.css";
+
+const Vote = ({ article }) => {
+  const [toggle, setToggle] = useState(false);
+  const [votes, setVotes] = useState(article.votes);
+
+  const handleClick = () => {
+    setToggle(!toggle);
+    let voteValue = 0;
+    if (toggle) {
+      voteValue = -1;
+    } else {
+      voteValue = 1;
+    }
+    axios
```

Comment:
Your votes aren't being rendered optimistically? The toggle is! But the votes aren't. Please correct this (currently the heart goes red before the vote changes which is a bit strange)